src/collection/components/modals/DeleteCollectionModal.tsx

```diff
@@ -57,12 +63,29 @@ const DeleteCollectionModal: FunctionComponent<DeleteCollectionModalProps> = ({
 		);
 	};
 
+	const renderDeleteMessageParagraph = () => {
+		if (isSharedWithOthers) {
+			return tText(
+				'collection/components/modals/delete-collection-modal___ben-je-zeker-dat-je-jezelf-van-deze-collectie-wil-wissen-deze-opdracht-is-met-count-andere-mensen-gedeeld-deze-verliezen-dan-toegang',
+				{ count: contributorCount }
+			);
+		}
+
+		if (isContributor) {
+			return tText(
+				'collection/components/modals/delete-collection-modal___ben-je-zeker-dat-je-jezelf-van-deze-collectie-wil-wissen'
+			);
+		}
+
+		return tText(
+			'collection/components/modals/delete-collection-modal___ben-je-zeker-dat-je-deze-collectie-wil-verwijderen'
+		);
+	};
```

Comment:
If it is possible to render a ReactNode with tHtml, we always prefer to use that over tText. Since meemoo might want to add html to render a link in the message. Only if the component doesn't accept a ReactNode input should we use tText. eg: for the placeholder of a textinput or the title field of a button.
```suggestion
	const renderDeleteMessageParagraph = () => {
		if (isSharedWithOthers) {
			return tHtml(
				'collection/components/modals/delete-collection-modal___ben-je-zeker-dat-je-jezelf-van-deze-collectie-wil-wissen-deze-opdracht-is-met-count-andere-mensen-gedeeld-deze-verliezen-dan-toegang',
				{ count: contributorCount }
			);
		}

		if (isContributor) {
			return tHtml(
				'collection/components/modals/delete-collection-modal___ben-je-zeker-dat-je-jezelf-van-deze-collectie-wil-wissen'
			);
		}

		return tHtml(
			'collection/components/modals/delete-collection-modal___ben-je-zeker-dat-je-deze-collectie-wil-verwijderen'
		);
	};
```

Reply:
I will adjust this in another PR where i added a confirm modal to simplify the component.